app/views/mail/newUser.scala.html

```diff
@@ -6,14 +6,12 @@
 
   @if(!enableAutoVerify) {
     <p>
-      Welcome to your new webKnossos account. You are almost ready to start exploring and to collaboratively
-      annotate large 3D datasets.
+      Welcome to your new webKnossos account. You are almost ready to share and annotate your large 3D datasets.
     </p>
     <p>Your account is currently under review. An administrator will activate you soon.</p>
   } else {
     <p>
-      Welcome to your new webKnossos account. Get ready to start exploring and to collaboratively
-      annotate large 3D datasets.
+      Welcome to your new webKnossos account. Get ready to share and annotate your large 3D datasets.
```

Comment:
This email doesn't seem to contain a link to wk. Might be a good idea to add this? Something similar to ` There are <a href="https://webknossos.org/publications">many published datasets</a> from the scientific community ready to be explored.` maybe..

Reply:
This email is for the self-hosted instances. I don't think it should direct users to wk.org. What do you think?

Reply:
Ah, good point, then it's alright for me.